roles/ampd/templates/testnet/config.toml.j2

```diff
@@ -68,15 +68,13 @@ chain_finalization = "RPCFinalizedBlock"
 type = "EvmVerifierSetVerifier"
 
 [[handlers]]
-chain_name = "hedera"
-chain_rpc_url = "{{ vault.testnet.ampd.hedera.rpc_url }}"
+chain_name = "sui"
+chain_rpc_url = "{{ vault.testnet.ampd.sui.rpc_url }}"
 cosmwasm_contract = "axelar1ce9rcvw8htpwukc048z9kqmyk5zz52d5a7zqn9xlq2pg0mxul9mqxlx2cq"
```

Comment:
This should be axelar1sykyha8kzf35kc5hplqk76kdufntjn6w45ntwlevwxp74dqr3rvsq7fazh

https://discord.com/channels/770814806105128977/1320786409778315355/1327353097445572619